homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
I like your work Sima :) however I don't understand why did you replace * with _ ? I am not sure this result still 12240.

Reply:
Thanks, It is is because of VS code formatting but I have sorted it.